src/lazy/binary/raw/v1_1/value.rs

```diff
@@ -292,6 +292,11 @@ impl<'top> LazyRawValue<'top, BinaryEncoding_1_1> for &'top LazyRawBinaryValue_1
             ..**self
         })
     }
+
+    #[cfg(feature = "source-location")]
+    fn location(&self) -> (usize, usize) {
```

Comment:
Do we have a `RowColumn` struct? (IIRC, it came up in one of the previous PRs.) If we do, we should use it here.

Reply:
No. I think it came in the design discussion though. I will add something for storing this.

Reply:
It's fine if it's literally just one of these:
```
pub struct RowColumn(pub usize, pub usize);
// or
pub type RowColumn = (usize, usize);
```
I know we don't normally make fields public, but in this case we're not really trying to define a new struct, we're just giving a descriptive name to `(usize, usize)`. For that same reason, I don't think it really needs any `impl` blocks.

I don't know which one of these two is better in this case. @zslayton, do you have any thoughts on the matter?